ui/dashboards/src/components/Variables/TemplateVariable.tsx

```diff
@@ -104,7 +104,7 @@ function ListVariable({ name }: TemplateVariableProps) {
         <Select
```

Comment:
I renamed this file while I was in here because this is the third or fourth time I've been confused because the file name doesn't match the exported component.

Reply:
Ah that was my bad during implementation. 

We meant for everything to be "Variable" and not "TemplateVariable". The idea was that Variables are more first class in Perses. Meaning they are explicitly supported in certain parts of the application rather than anywhere in a templatized string (unlike Grafana).

I think they're basically interchangeable but for sake of consistency with components like `VariableList` and the Dashboard spec, maybe it's best to remove anything that prefixes "Template" and renaming it to "Variable". However, I realize that this will result in a few breaking changes, such as the `TemplateVariableProvider` and `TemplateVariable` components. (Not for this PR)

Reply:
Ah gotcha. I'll leave this for file/component consistency for now, and we can take a pass at s/TemplateVariable/Variable in a future PR.